nixos/doc/manual/configuration/linux-kernel.chapter.md

```diff
@@ -92,6 +92,24 @@ To use your custom kernel package in your NixOS configuration, set
 boot.kernelPackages = pkgs.linuxPackagesFor yourCustomKernel;
 ```
 
+## Rust
+
+The Linux kernel does not have Rust language support enabled by
+default. For kernels newer than 6.6, experimental Rust support can be
```

Comment:
> For kernels newer than 6.6

Based on the test, this actually should read "For kernel versions 6.6 or newer,"